versioned_docs/version-2.x/advanced/server/add-http-endpoint.mdx

```diff
@@ -181,3 +181,42 @@ export default {
   },
 };
 ```
+
+## Extend the HTTP server with `onServerInit`
+
+<SinceVersion tag="2.30" />
+
+The `onServerInit` hook is a server module extension point introduced in
+Front-Commerce 2.30. It allows you to extend the Express HTTP server or perform
+additional tasks during server initialization.
+
+To use the `onServerInit` hook, follow these steps:
+
+1. Create a new server module or open an existing one.
```

Comment:
It could be good to link to the server module docs.

```suggestion
1. Create a new [server module](/docs/2.x/advanced/server/add-http-endpoint#what-is-a-module) or open an existing one.
```

Reply:
Updated with your suggestion